app/src/lib/menu-update.ts

```diff
@@ -430,6 +431,23 @@ function getNoRepositoriesBuilder(state: IAppState): MenuStateBuilder {
   return menuStateBuilder
 }
 
+function getAppMenuBuilder(): MenuStateBuilder {
+  const menuStateBuilder = new MenuStateBuilder()
+
+  const isResizableActive =
+    document.activeElement?.closest('.resizable-component') !== null
+  menuStateBuilder.setEnabled(
```

Comment:
Why does this crash the app? I must be completely overlooking something obvious...

Reply:
It says 'increase-active-resizable-width' is an unknown id?

Reply:
Yep it was something obvious. I had copied the zoom/zoom out  menu item entries for the menu directly above.. which we don't ever disable.. If you want to be able to "id"entify one of these you need to give it an "id". 🤦